subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -67,235 +67,134 @@
                 <main class="main-content">
                     <nav class="docs-navigation">
                         <ul>
-                            <li><a href="/userguide/userguide.html">User Manual Home</a></li>
-                            <li><a href="/release-notes.html">Release Notes</a></li>
+                            <li>
+                                <a href="/userguide/userguide.html">User manual</a>
+                                <span style="font-size: 70%;">(
+                                    <a style="display: inline; font-size: 100%;" href="/userguide/userguide_single.html">single page</a>
+                                    |
+                                    <a style="display: inline; font-size: 100%;" href="/userguide/userguide.pdf">PDF</a>)
+                                </span>
+                            </li>
+                            <li><a href="/release-notes.html">Release notes</a></li>
+                            <li><a href="/userguide/installation.html">Installing Gradle</a></li>
                         </ul>
 
-                        <h3 id="getting-started">Getting Started</h3>
+                        <h3 id="tutorials"><a href="https://guides.gradle.org/">Tutorials</a></h3>
```

Comment:
❓ is there a better way to visually distinguish between a header with subcontent and a prominent link (using a header)?

We have 3 places where we link off to different areas of docs: the top nav, the side nav (dispersed), and the footer. I think there is opportunity here to do a better job by unifying the top nav and side nav for this.  Do you have any specific ideas for how to achieve this?

Reply:
I thought we'd agreed to remove the dropdown from the top nav and just leave it as a "Docs" link. We discussed it at least. That's still my preferred option.

I don't really see the value of the documentation links in the footer. Better would be a persistent link to the top of the page when the page is scrolled down. Alternatively, the nav could persist in the view while the page scrolls. I don't know how these options are implemented, so I don't know what the pros and cons are.

Reply:
As for visually distinguishing between headers and main links, do we have access to a designer that could offer suggestions?

Reply:
I have updated the look of all the "main" links in the left nav. I also moved the third-party integrations page link to the Reference section. It was like an out-of-place orphan before.